crosshair/objectproxy.py

```diff
@@ -2,6 +2,11 @@
 import operator
 import sys
 
+try:
+    long
+except NameError:
+    long = int
```

Comment:
"long" is later used in the definition for the `__long__` method, which I think can also be removed. Would you mind removing that method too?